GroupProject/Pages/Index.cshtml.cs

```diff
@@ -23,42 +23,42 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public void OnGet()
         {
-            using(var webClient = new WebClient())
+            using (var webClient = new WebClient())
             {
-                IDictionary<long, QuickTypePlayerDetails.PlayerDetail> allplayers = new Dictionary<long, QuickTypePlayerDetails.PlayerDetail>();
-                String playersJSON = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/CompetitionDetails/EPL?key=bc49021bad1943008414c5a75e665961");
-                QuickTypePlayerDetails.PlayerDetail[] playerDetails = QuickTypePlayerDetails.PlayerDetail.FromJson(playersJSON);
-                
+                //IDictionary<long, QuickTypePlayerDetails.PlayerDetail> allplayers = new Dictionary<long, QuickTypePlayerDetails.PlayerDetail>();
+                //String playersJSON = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/CompetitionDetails/EPL?key=bc49021bad1943008414c5a75e665961");
+                //QuickTypePlayerDetails.PlayerDetail[] playerDetails = QuickTypePlayerDetails.PlayerDetail.FromJson(playersJSON);
+                //foreach (QuickTypePlayerDetails.PlayerDetail player in playerDetails)
+                //{
+                //    allplayers.Add(player.PlayerId, player);
+                //}
+
 
-                foreach (QuickTypePlayerDetails.PlayerDetail player in playerDetails)
-                {
-                    allplayers.Add(player.PlayerId, player);
-                }
 
                 string Membership = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
                 JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("PlayerInfoSchema.json"));
-                JArray jsonObject = JArray.Parse(Membership);
+                JArray jarray = JArray.Parse(Membership);
```

Comment:
Give a more descriptive name.  jarray is just repeating the variable type.  We couldn't do that, or we'd have hundreds of variables called "string".   Consider a name that is descriptive of what this variable does.